src/Nethermind/Nethermind.JsonRpc/JsonRpcProcessor.cs

```diff
@@ -137,9 +137,7 @@ JsonRpcResult GetParsingError(string error, Exception? exception = null)
         try
         {
             // Asynchronously reads data from the PipeReader.
-            ReadResult readResult = _jsonRpcConfig.MaxRequestBodySize is not null
-                ? await reader.ReadAtLeastAsync((int)_jsonRpcConfig.MaxRequestBodySize.Value)
-                : await reader.ReadToEndAsync();
```

Comment:
I'm torn about this one, @benaadams ?

Reply:
Am wondering about adding a timeout also?

Reply:
vs a very slow request; like if the full request takes more than 1sec to receive chop it?

Reply:
We have some global timeouts on json rpc

Reply:
Will think about it a bit more.